app/assets/javascripts/select-kit/addon/components/composer-actions.js

```diff
@@ -95,7 +99,9 @@ export default DropdownSelectBoxComponent.extend({
     return {};
   },
 
-  content: computed("seq", function () {
+  @discourseComputed("seq")
+  // eslint-disable-next-line no-unused-vars
+  content(seq) {
```

Comment:
you don't need the seq here, it's only needed if you are going to use it in function body

Reply:
Yeah, we don't use it directly but unfortunately, we still need it here.  The method uses through `this` a lot of stuff that should be passed as an argument of computed property.  The only reason we have `seq` in this component is to make sure that UI updates correctly. 

It was introduced [here](https://github.com/discourse/discourse/commit/fcc5ef9625069e7a33a7ae75063ae5de38fc06e9). It isn't an ideal solution but it works 🙂 . I would fix it in the future but it needs a careful approach 🙂 .